lib/chef/resource/mount.rb

```diff
@@ -40,7 +40,7 @@ class Mount < Chef::Resource
         description: "Windows only:. Use to specify the password for username.",
         sensitive: true
 
-      property :mount_point, String, name_property: true,
+      property :mount_point, String, name_property: true, identity: false,
```

Comment:
why does this one become false?

Reply:
left over edit from an earlier version of the patch.  removed.